scripts/automator/packages/manager-dnf

```diff
@@ -1,2 +1,3 @@
 # Package repo: https://apps.fedoraproject.org/packages/
-packages+=(ccache libtool ncurses-devel SDL2-devel SDL2_net-devel opusfile-devel glib2-devel)
+packages+=(ccache make cmake libpng-devel libtool ncurses-devel
```

Comment:
`glib2-devel`, `cmake`, and `libtool` are not dosbox-staging dependencies.

`automake` and `alsa-lib-devel` are missing.

Reply:
Right; `glib2-devel, cmake` are dependencies needed to build FluidSynth from source.  

In cases where we can't install FS 2.x directly from the repo, we instead installing the dependencies to build it (if possible) using contrib.

Reply:
> automake and alsa-lib-devel are missing.

From a vanilla install, automake was pulled in by one of the other packages. Likewise, I didn't have to install alsa libs, and sound output works fine.

Reply:
Not complaining, just informing - changes are not harmful in any way, so I have no problem with merging them in :)

> In cases where we can't install FS 2.x directly from the repo, we instead install the dependencies to build it (if possible) using contrib.

For cases when someone wants to test if dosbox-staging+FS2.x builds on F32 or older (without testing) -> use `toolbox` to create local Fedora 33 container and build inside that:
```
$ cat /etc/os-release  | head -n2
NAME=Fedora
VERSION="32 (Workstation Edition)"
$ toolbox create --release 33
$ toolbox enter --release 33
$ cat /etc/os-release | head -n2
NAME=Fedora
VERSION="33 (Container Image Prerelease)"
```
Fedora 33 went to beta recently (can be installed already), and is on track for release in 15 days, so in around 1 month, 60-70% of Fedora users won't need to build FS 2.x themselves at all :) By the time Debian Stable images will be released, F32 should reach EOL, so we'll cover ~98% of users.

> Likewise, I didn't have to install alsa libs, and sound output works fine.

You definitely needed to install `alsa-lib-devel` (those are headers, no headers are installed by default). Perhaps some other software pulled it in as well (but clean `toolbox` container won't have it).

But this discussion reminded me about Fedora Silverblue… they would prefer Flatpak package… perhaps I'll take a stab at submitting one to Flathub after all.